Libraries/Components/HyperlinkWindows/HyperlinkWindows.windows.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * All rights reserved.
+ *
+ * This source code is licensed under the MIT-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @providesModule HyperlinkWindows
+ */
+'use strict';
+
+var PropTypes = require('prop-types');
+var React = require('React');
+var ReactNativeViewAttributes = require('ReactNativeViewAttributes');
+var ViewPropTypes = require('ViewPropTypes');
+var requireNativeComponent = require('requireNativeComponent');
+
+const ColorPropType = require('ColorPropType');
+const EdgeInsetsPropType = require('EdgeInsetsPropType');
+const StyleSheetPropType = require('StyleSheetPropType');
+const Text = require('Text');
+const TextStylePropTypes = require('TextStylePropTypes');
+const TouchableWithoutFeedback = require('TouchableWithoutFeedback');
+
+const stylePropType = StyleSheetPropType(TextStylePropTypes);
+
+class HyperlinkWindows extends React.Component {
+  props: {
+    ellipsizeMode: 'head' | 'middle' | 'tail' | 'clip',
+    numberOfLines: number,
+    textBreakStrategy: 'simple' | 'highQuality' | 'balanced',
+    onLayout: Function,
+    onPress: Function,
+    onLongPress: Function,
+    pressRetentionOffset: EdgeInsetsPropType,
+    selectable: boolean,
+    selectionColor: ColorPropType,
+    suppressHighlighting: boolean,
+    style: stylePropType,
+    testID: string,
+    nativeID: string,
+    allowFontScaling: boolean,
+    accessible: boolean,
+    adjustsFontSizeToFit: boolean,
+    minimumFontScale: number,
+    disabled: boolean,
+  };
+
+  static propTypes = {
+    ...Text.PropTypes,
+  };
+
+  static defaultProps = {
+  };
+
+  render() {
+    return (
+        <TouchableWithoutFeedback 
```

Comment:
@reseul This will result in canvas wrapping the hyperlink? If so will this properly work for accessibility in the RichTextBlock?

Reply:
@reseul - I had the same question, looking through `TouchableWithoutFeedback`, I don't see where it wraps the child in a `<View/>` component though...

---
In reply to: [182900320](https://github.com/Microsoft/react-native-windows/pull/1794#discussion_r182900320) [](ancestors = 182900320)

Reply:
No, this just is a way to inject the Touchable mix into any React component.  It shouldn't wrap the hyperlink with any XAML components.

However, as Radu pointed out, there is an issue with getting keyboard input.  I am looking into this and will get a revision out soon.